wiki/Beatmap_information/en.md

```diff
@@ -0,0 +1,141 @@
+
+The nominator section is added as soon as the beatmap is nominated by [beatmap nominators](/wiki/People/Beatmap_Nominators) for becoming ranked.
+
+After the beatmap is ranked, users are able to vote for [user tags](/wiki/Beatmap/Beatmap_tags#user-tags). Upon reaching 5 votes on an user tag, the respective tag will be displayed in the `user tag` segment in this section.
+
+### Success rate display
+
+![](img/success-rate.png "Success rate display")
+
+The success rate display visualises how many players were able to successfully pass the beatmap. When hovering over the percentage value, a tooltip will provide the exact number of players.
+
+Below follows the points of failure, a diagram showing at what points users failed their play. A higher spike means that more players failed at that point.
+
+## Hype train
+
+If a beatmap is WIP or pending, the hype train section is visible where users can [hype](/wiki/Beatmap/Hype) the beatmap to indicate that they want the beatmap to be ranked.
+
+## Rankings
+
+![](img/rankings.png "Global ranking on a beatmap and the top score")
+
+If a beatmap is qualified or ranked, it has access to leaderboards, on which players can compete against each other.
+
+osu! offers the 3 leaderboards `Global Ranking`, `Country Ranking`, and `Friend Ranking`, of which the last two require [osu!supporter](/wiki/osu!supporter) for usage. Selecting one of the tabs at the top of the menu will display the respective leaderboard in the score listing below. Clicking on one or several mod icons in the filter menu allows the user to show solely scores with that exact [mod](/wiki/Gameplay/Game_modifier) combination.
+
+The score listing shows up to 50 top scores, while the best score is highlighted specifically with an enlarged score card.
+
+Hovering over a score will reveal 3 dots at the right side of the menu. Clicking on the dots opens up a menu with 3 options:
+
+- **View details:** The score will be visualised on a single page.
+- **Download replay:** Download the [replay](/wiki/Gameplay/Replay) file.
+- **Report score:** If users believe that a score is impossible to achieve without using cheats, they are able to report a score for further investigation.
+
+## Comments
+
+![](img/comments.png "Textbox for posting comments and sorting options")
+
+In the comment section, users are able to exchange their opinions about the beatmap. The beatmap host also has the ability to pin a comment, which will cause it to be displayed at the top of the section. The number at the top reflects the number of total comments. Comments can be sorted by either most recent, oldest, or comments with the most upvotes by selecting the respective sort mode below the input box. To publish a new comment, write something in the textbox, and click `Post` or press `Enter`.
+
+When clicking the `Watch` button on the right side, the user receives notifications about new comments on this beatmap.
```

Comment:
More appropriate tense. They won't receive notis immediatedly, but rather as they come from that moment onwards.
```suggestion
When clicking the `Watch` button on the right side, the user will receive notifications about new comments on this beatmap.
```